code/service/src/main/resources/application.conf

```diff
@@ -367,26 +367,36 @@ chiefofstate {
 		propagated-headers = ""
 		propagated-headers = ${?COS_WRITE_SIDE_PROPAGATED_HEADERS}
 	}
-	
+
 	read-side {
 		# set this value to true whenever a readSide config is set
 		enabled = false
 		enabled = ${?COS_READ_SIDE_ENABLED}
 	}
-	
+
 	plugin-settings {
 		enabled-plugins = "com.namely.chiefofstate.plugin.PersistedHeaders"
 		enabled-plugins = ${?COS_ENABLED_PLUGINS}
 	}
-	
+
 	telemetry {
 		namespace = ""
 		namespace = ${?COS_TELEMETRY_NAMESPACE}
-		
+
 		otlp_endpoint = ""
 		otlp_endpoint = ${?COS_TELEMETRY_COLLECTOR_ENDPOINT}
-		
+
 		trace_propagators = "b3multi"
 		trace_propagators = ${?COS_TRACE_PROPAGATORS}
 	}
 }
+
+cos-migration {
```

Comment:
This setting should go under chief-of-state section even though it is temporary.